### PR TITLE
Feature/ietf007 iab site updates fed

### PR DESCRIPTION
**Pull request by CliffRobinson**
Front-end tweaks for IAB

### CODE REVIEW

File: ietf/static_src/css/iab.scss [hunk 1]
```diff
@@ -158,6 +162,12 @@
     }
 }
 
+@include media-breakpoint-up(xl) {
+    .nav-item--iab {
+        max-width: none;
+    }
+}
+
```
Comment (liamjohnston) [inline ietf/static_src/css/iab.scss]: Media queries should go inside the selector, i.e.
```
.nav-item--iab {
  [styles]
  
  @include media-breakpoint-up(xl) {
     max-width: none;
   }
}
```

Comment (liamjohnston) [reply ietf/static_src/css/iab.scss]: (when using sass, which you are)

File: ietf/static_src/css/iab.scss [hunk 2]
```diff
@@ -171,3 +181,34 @@
 .iab_body {
     background-color: rgba(62, 139, 193, 0.08);
 }
+
+.dropdown-toggle__iab {
+    &:after {
```
Comment (liamjohnston) [inline ietf/static_src/css/iab.scss]: Minor point – it works either way, but _technically_ pseudo elements (before & after) should have double colon, i.e. `::after`

Comment (liamjohnston) [reply ietf/static_src/css/iab.scss]: Also shouldn't this be a modifier not an element? i.e. `.dropdown-toggle--iab`

File: ietf/static_src/css/iab.scss [hunk 3]
```diff
@@ -171,3 +181,34 @@
...
+}
+
+.iab-tabs > li > :not(.active) {
+    color: #616466 !important;
+    border-bottom: 2px solid var(--color-iab-grey-5) !important;
+
+    &:hover {
+        color: var(--color-iab-navy) !important;
+    }
+}
```
Comment (liamjohnston) [inline ietf/static_src/css/iab.scss]: Beware the specificity creep here. I see there's even more `!important`s being added. With BEM usually we can rely on a flat specificity graph (everything stylable has a class) and source order, where modifiers are declared after the base class.

Should you do e.g. 
- `<li class="iab-tabs__item">`
- `<a href="..." class="iab-tabs__link">`
- `<a href="..." class="iab-tabs__link active">` (or `<a href="..." class="iab-tabs__link iab-tabs__link--active">`)

Something to think about if, you haven't already. I know this project uses bootstrap (i.e. `active` may be baked in), so we can't necessarily do the typical BEM things..

File: ietf/static_src/css/iab.scss [hunk 4]
```diff
@@ -171,3 +181,34 @@
...
+    text-align: left;
+}
+
+.iab-tabs > li > .active {
+    color: var(--color-iab-navy) !important;
+    border-bottom: 2px solid black !important;
+}
+
+.iab-tabs > li > :not(.active) {
+    color: #616466 !important;
```
Comment (liamjohnston) [inline ietf/static_src/css/iab.scss]: Do we not have a colour variable to use?

File: ietf/static_src/css/iab.scss (hunk 1 above)
Comment (CliffRobinson) [reply ietf/static_src/css/iab.scss]: Thanks, I figured there was some better syntax for this

File: ietf/static_src/css/iab.scss (hunk 2 above)
Comment (CliffRobinson) [reply ietf/static_src/css/iab.scss]: Yep, I've changed this

File: ietf/static_src/css/iab.scss (hunk 4 above)
Comment (CliffRobinson) [reply ietf/static_src/css/iab.scss]: We didn't, but now we do

File: ietf/static_src/css/iab.scss (hunk 3 above)
Comment (CliffRobinson) [reply ietf/static_src/css/iab.scss]: Yep, trying to customise something bootstrap didn't really contemplate customising lead to more `!important`s than I wanted. I've given it another going over and removed some that didn't need to be there, but yeah, I don't think we can do perfect BEM and keep the client's requirement for bootstrap in this case.

**Review by liamjohnston — APPROVED** [review]
cool beans

File: ietf/static_src/css/iab.scss [hunk 5]
```diff
@@ -171,3 +179,34 @@
...
+}
+
+.iab-tabs > li > a {
+    padding: rem(16) rem(18) rem(10);
+    text-align: left;
+}
+
+.iab-tabs > li > .active {
+    color: var(--color-iab-navy) !important;
+    border-bottom: 2px solid black !important;
```
Comment (Juliet-Brown) [inline ietf/static_src/css/iab.scss:202]: Anyway around it we could avoid` importants`?

File: ietf/static_src/css/iab.scss [hunk 6]
```diff
@@ -55,10 +55,12 @@
     background-color: var(--color-iab-navy) !important;
 }
 
+.iab-logo-image {
+    width: 100px !important;
+}
```
Comment (Juliet-Brown) [inline ietf/static_src/css/iab.scss:60]: If your writing custom css I would suggest BEM  In the future :)

**Pull request merged**